api/base/serializers.py

```diff
@@ -1,16 +1,33 @@
 import re
 import collections
 
-from rest_framework.reverse import reverse
 from rest_framework.fields import SkipField
+from rest_framework.exceptions import NotFound
+from rest_framework.reverse import reverse
 from rest_framework import serializers as ser
 
+from framework.auth.core import Auth
+from framework.guid.model import Guid
 from website import settings
+from website.project.model import Node, Comment
 from website.util.sanitize import strip_html
 from website.util import waterbutler_api_url_for
```

Comment:
Not sure how zealous we are about this but our [style-guide](http://cosdev.readthedocs.org/en/latest/style_guides/python.html#imports) says that these imports should be something more like `from framework.auth import core` and then `new_auth = core.Auth()`. If it's not a thing we do then we should just create an issue to update the style guide and I'll stop noting it in code reviews ;)